src/auth/auth.service.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  ConflictException,
+  Injectable,
+  Logger,
+  NotFoundException,
+} from '@nestjs/common';
+import { AuthDto } from './dto/auth.dto';
+import { PrismaService } from 'src/prisma.service';
+import * as bcrypt from 'bcryptjs';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class AuthService {
+  logger = new Logger(AuthService.name);
+  constructor(
+    private primaService: PrismaService,
+    private jwtService: JwtService,
+  ) {}
+
+  async signin(body: AuthDto) {
+    try {
+      const user = await this.primaService.user.findUnique({
+        where: {
+          email: body.email,
+        },
+      });
```

Comment:
I think this logic should be handled in UsersService, here we will import UsersService and use its function